Data Products (#745)
snowplow/documentation: docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/ui/index.md

---
title: "Managing Data Product using Console"
date: "2024-01-18"
sidebar_label: "Using the UI"
sidebar_position: 1
sidebar_custom_props:
  offerings:
    - enterprise
    - cloud

---

## Creating and editing a new Data Product using Console

To create a new data product, navigate to the "Data products" option in the left main menu and click the "Create data product" button. 

![Create data product](images/image-1.png)

A form will appear on the page. Enter your data product information and click "Create and continue" to navigate to the event specification page.

![Event specifications](images/image-2.png)

> This page allows you to create multiple event specifications. You can click on any row to enter the details on this screen, or you can complete the information later.

Author: done

> When clicking on a event specification row, a page will allow you to enter additional information into separate modals:

Reviewer: Should we rephrase `When clicking on a event specification row, a page will allow you to enter additional information into separate modals:` as `Clicking on an event specification allows you to enter further details for the following:`

Author: done

- **Event information**; describes information such as which applications the event fires in
- **Event data structure**; determines how your data is structured and define the types of properties of the event
- **Entity data structures**; describes which entities to attach to the event when it is triggered
- **Event triggers**; specify the places and circumstances under which the event is triggered

> - **Implementation instructions**; specify instructions about how each schema property should be set for this scenario

Reviewer: `Implementation instructions` has now been changed to `Properties` so we should update it here. Also we should replace `scenario` with `event specification`.

Author: done

> You can navigate to an overview of a data product by selecting its name in the breadcrumb navigation or by selecting in the data product listing always visible in the left menu.

Reviewer: ```suggestion
The breadcrumb navigation allows you to quickly navigate to the data product overview as well as to the list of data products. Alternatively, you can access the list of available data products by clicking `Data products` prominently displayed in the left menu.
```

> The breadcrumb navigation allows you to quickly navigate to the data product overview as well as to the list of data products. Alternatively, you can access the list of available data products by clicking `Data products` prominently displayed in the left menu.

Reviewer 2: ```suggestion
The breadcrumb navigation allows you to quickly navigate to the data product overview as well as to the list of data products. Alternatively, you can access the list of available data products by clicking `Data products` prominently displayed in the navigation bar on the left.
```

> In the image below, you can see an example of a data product. It not only provides an overview of all the event specifications but also allows you to access three important functionalities.

Reviewer 2: ```suggestion
In the image below, you can see an example of a data product. It not only provides an overview of all the event specifications but also allows you to access three important pieces of functionality.
```

> - **Share**; allows other members of your organization to access the data product 

Reviewer: ```suggestion
- **Share**; allow other members of your organization to access the data product 
```

Author: done

> - **Subscribe**; allow other members to being notified of any changes in the data product

Reviewer: ```suggestion
- **Subscribe**; receive notifications of any changes in the data product
```

> - **Implement tracking**; allow to automatically generate the code for your data product to be included in your application (to learn more visit [Code Generation - automatically generate code for Snowplow tracking SDKs](/docs/collecting-data/code-generation/index.md) 

Reviewer: ```suggestion
- **Implement tracking**; automatically generate the code for your data product to be included in your application (to learn more visit [Code Generation - automatically generate code for Snowplow tracking SDKs](/docs/collecting-data/code-generation/index.md) 
```

> - **Implement tracking**; automatically generate the code for your data product to be included in your application (to learn more visit [Code Generation - automatically generate code for Snowplow tracking SDKs](/docs/collecting-data/code-generation/index.md) 

Reviewer 2: ```suggestion
- **Implement tracking**; automatically generate the code for your data product to be included in your application (to learn more visit [Code Generation - automatically generate code for Snowplow tracking SDKs](/docs/collecting-data/code-generation/index.md)) 
```

> *Notes: share and subscribe are allowed only for users registered in Snowplow BDP Console.*

Reviewer: ```suggestion
*Notes: sharing and subscribing is only available for users registered in Snowplow BDP Console.*
```

> *Notes: sharing and subscribing is only available for users registered in Snowplow BDP Console.*

Reviewer 2: Is that true? Did we not agree to leave it open as long as it's a valid email address?

---
title: "Managing Data Product using Console"
date: "2024-01-18"
sidebar_label: "Using the UI"
sidebar_position: 1
sidebar_custom_props:
  offerings:
    - enterprise
    - cloud

---

## Creating and editing a new Data Product using Console

To create a new data product, navigate to the "Data products" option in the left main menu and click the "Create data product" button. 

![Create data product](images/image-1.png)

A form will appear on the page. Please enter your data product information into the form.

> Cliking on the "Create and continue" will brig you to the definition of event specifications page:

Reviewer: A few typos here. Should this instead say something along the lines of `Enter your data product information and click "Create and continue" to navigate to the event specification page` to avoid some repetition?

Author: done good suggestion

> ![Event specifications](images/image-2.png)

Author: done thanks
